protobuf/src/main/java/io/grpc/protobuf/ProtoUtils.java

```diff
@@ -78,9 +79,20 @@
    */
   @ExperimentalApi("https://github.com/grpc/grpc-java/issues/1786")
   public static <T extends Message> Marshaller<T> jsonMarshaller(final T defaultInstance) {
+    final Parser parser = JsonFormat.parser();
     final Printer printer = JsonFormat.printer();
-    // TODO(carl-mastrangelo): Add support for ExtensionRegistry (TypeRegistry?)
```

Comment:
FYI: We could probably add support by using ProtoLiteutils.globalRegistry, but there's no getter now and we'd maybe need to flesh that out a bit more.